Mathlib/Data/Finsupp/Basic.lean

```diff
@@ -719,101 +715,6 @@ end FInjective
 
 end ComapDomain
 
-/-! ### Declarations about finitely supported functions whose support is an `Option` type -/
-
-
-section Option
-
-/-- Restrict a finitely supported function on `Option α` to a finitely supported function on `α`. -/
-def some [Zero M] (f : Option α →₀ M) : α →₀ M :=
-  f.comapDomain Option.some fun _ => by simp
```

Comment:
If this code is moving, could you move it first in a standalone PR?

Reply:
Good point

Reply:
Ok https://github.com/leanprover-community/mathlib4/pull/26309 is merged